en/docs/edit-the-code/intellisense.md

```diff
@@ -1 +1,44 @@
-# IntelliSense
\ No newline at end of file
+# IntelliSense
+
+### Code Completion & Snippets
+The extension provides you with suggestions on variables, keywords, and code snippets of language constructs. (such as functions, type definitions, services, iterable constructs, etc.)
+
+<img src="https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/blob/main/en/docs/img/code-completion.gif?raw=true" width="100%" />
+
+Following are some key features of the code completion support provided by the plugin. 
+
+#### Context aware completion items
+The completion item list is sorted based on the context of the current cursor position.
+
+<img src="https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/blob/main/en/docs/img/context-aware-completions.gif?raw=true" width="100%" />
```

Comment:
The recording in the gif seems a bit slow. i.e., waiting too much time on the appeared menu. Can we fix that?

Reply:
Done